Configure

```diff
@@ -240,6 +240,7 @@ my @disablables = (
     "autoalginit",
     "autoerrinit",
     "bf",
+    "bp",
```

Comment:
PAIR_GROUP or EC_BP_GROUP? Though PAIRING_GROUP isn't longer than, say, X509_STORE_CTX, and imho is the most obvious.